### ISSUE TITLE
Undefined symbols for architecture armv7: "GULAppEnvironmentUtil"

### ISSUE THREAD

**Issue by SotoiGhost** [firebase/firebase-ios-sdk#5282]
### [REQUIRED] Step 1: Describe your environment

  * Xcode version: 11.3.1
  * Firebase SDK version: 6.21.0
  * Firebase Component: InAppMessaging
  * Component version: 0.19.1
  * Installation method: `CocoaPods` (`cocoapods-rome` plugin)

### [REQUIRED] Step 2: Describe the problem

When building the framework I get the following error:

```
Undefined symbols for architecture armv7:
  "_OBJC_CLASS_$_GULAppEnvironmentUtil", referenced from:
      objc-class-ref in FIRInAppMessaging+Bootstrap.o
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

#### Steps to reproduce:

What happened? How can we make the problem occur?
Unzip the file and run `make` (`cocoapods-rome` plugin must be installed to build).

[InAppMessaging.zip](https://github.com/firebase/firebase-ios-sdk/files/4429945/InAppMessaging.zip)

**Comment by SotoiGhost** [comment]
Workaround for now. Add these lines to your Podfile file:

```
post_install do |installer|
	installer.pods_project.targets.each do |pod|
		if pod.name == "FirebaseInAppMessaging"
			pod.build_configurations.each do |config|
				if config.name == 'Release'
					puts "Linking missing 'GoogleUtilities' framework to #{pod.name}"
					config.build_settings['OTHER_LDFLAGS'] ||= ['$(inherited)','-framework "GoogleUtilities"']
				end
			end
		end
	end
end
```

**Comment by paulb777** [comment]
Thanks for sending the fix and workaround along with  the bug!

It's merged to master and is targeted for the M69-6.23.0 release.
